components/dashboard/src/components/BillingAccountSelector.tsx

```diff
@@ -96,7 +96,17 @@ export function BillingAccountSelector(props: { onSelected?: () => void }) {
             {teamsAvailableForAttribution === undefined && <Spinner className="m-2 h-5 w-5 animate-spin" />}
             {teamsAvailableForAttribution && (
                 <div>
-                    <p>Associate usage without a project to the billing account below.</p>
+                    <h2 className="text-gray-500">
+                        Associate usage without a project to the billing account below.{" "}
+                        <a
+                            className="gp-link"
+                            href="https://www.gitpod.io/docs/configure/billing"
+                            target="_blank"
+                            rel="noreferrer"
+                        >
+                            Learn more
```

Comment:
**thought**: Added [a report](https://eu.mixpanel.com/s/1K13ak) (internal) to measure how usage (clicks) for these **help links** will change. Cc @jakobhero for visibility and in case we could track this better from inbound traffic to the docs.

Reply:
Unholding this PR for now. We can iterate with better solution in follow-up PRs. 🙏🏽